src/net_processing.cpp

```diff
@@ -245,6 +245,10 @@ struct Peer {
     double m_addr_token_bucket{1.0};
     /** When m_addr_token_bucket was last updated */
     std::chrono::microseconds m_addr_token_timestamp{GetTime<std::chrono::microseconds>()};
+    /** Total number of addresses processed. */
+    std::atomic<uint64_t> m_addr_processed{0};
```

Comment:
nit: the name "processed" is confusing because from the first glance it's unclear whether it includes limited or not

Reply:
Added more comments.